modules/nf-core/icountmini/peaks/tests/main.nf.test

```diff
@@ -0,0 +1,86 @@
+nextflow_process {
+
+    name "Test Process ICOUNTMINI_PEAKS"
+
+    script "../main.nf"
+    process "ICOUNTMINI_PEAKS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "icountmini"
+    tag "icountmini/segment"
+    tag "icountmini/sigxls"
+    tag "icountmini/peaks"
+
+    setup {
+        run("ICOUNTMINI_SEGMENT") {
+            script "../../segment/main.nf"
+            process {
+                """
+                input[0] = [
+                    [id:'test', single_end:false],// meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/chr21_gencode.gtf', checkIfExists:true)
+                ]
+                input[1] = file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai', checkIfExists:true)
+                """
+            }
+        }
+
+        run("ICOUNTMINI_SIGXLS") {
+            script "../../sigxls/main.nf"
+            process {
+                """
+                input[0] = [
+                    [id:'test'],// meta map
+                    file("https://raw.githubusercontent.com/nf-core/test-datasets/clipseq/crosslinks/clippy.bed", checkIfExists: true)
+                ]
+                input[1] = ICOUNTMINI_SEGMENT.out.gtf.flatten().last()
```

Comment:
This is a bit weird to need flatten and last?

Reply:
True, I don't know why they did so previously.
I will investigate and remove it if possible.

Reply:
A `.map { it[1] }` is sufficient